apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,31 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should see mark book as finished', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
```

Comment:
Again, why not put these in a beforeEach?

Reply:
Done, moved it to `beforeEach`.